@xen-orchestra/lite/src/views/pool/PoolNetworkView.vue

```diff
@@ -1,11 +1,200 @@
 <template>
-  <PageUnderConstruction />
+  <div class="pool-network-view">
+    <UiCard class="card">
+      <div class="networks-content">
+        <UiTitle>
+          {{ $t('network') }}
+          <template #actions>
+            <UiButton :left-icon="faPlus" variant="secondary" accent="info" size="medium">{{ $t('new') }}</UiButton>
+          </template>
+        </UiTitle>
+        <UiQuerySearchBar />
+        <UiTableActions title="Table actions">
+          <UiButton :left-icon="faEdit" variant="tertiary" accent="info" size="medium">{{ $t('edit') }}</UiButton>
+          <UiButton :left-icon="faTrash" variant="tertiary" accent="danger" size="medium">{{ $t('delete') }}</UiButton>
+        </UiTableActions>
+        <div>
+          <UiTopBottomTable
+            :selected-items="selectedItemsNetwork"
+            :total-items="reactiveNetworksWithVLANs.length"
+            @toggle-select-all="toggleSelectNetwork"
+          />
+          <VtsTable>
+            <thead>
+              <tr>
+                <th class="checkbox-header">
+                  <UiCheckbox accent="info" @update:model-value="toggleSelectNetwork($event)" />
+                </th>
+                <ColumnTitle id="name" :icon="faAlignLeft">{{ $t('name') }}</ColumnTitle>
+                <ColumnTitle id="description" :icon="faAlignLeft">{{ $t('description') }}</ColumnTitle>
+                <ColumnTitle id="pifs" :icon="faPowerOff">{{ $t('pifs-status') }}</ColumnTitle>
+                <ColumnTitle id="vlan" :icon="faAlignLeft">{{ $t('vlan') }}</ColumnTitle>
+                <ColumnTitle id="mtu" :icon="faHashtag">{{ $t('mtu') }}</ColumnTitle>
+                <ColumnTitle id="locking" :icon="faCaretDown">{{ $t('locking-mode') }}</ColumnTitle>
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(item, index) in reactiveNetworksWithVLANs" :key="index">
+                <td>
+                  <UiCheckbox v-model="item.selected" accent="info" />
+                </td>
+                <td>{{ item.name_label }}</td>
+                <td>{{ item.name_description }}</td>
+                <td>
+                  <!-- TODO improvement required -->
+                  <UiInfo v-if="item.status === 'connected'" accent="success"> {{ item.status }}</UiInfo>
+                  <UiInfo v-else-if="item.status === 'disconnected'" accent="danger"> {{ item.status }}</UiInfo>
+                  <UiInfo v-else accent="warning"> {{ item.status }}</UiInfo>
+                </td>
+                <td>{{ item.vlan }}</td>
+                <td>{{ item.MTU }}</td>
+                <td>{{ item.default_locking_mode }}</td>
+              </tr>
```

Comment:
This `<tr>` can be extracted into a child component, for example, a `PifRow.vue` component, that will accept a PIF object as a prop and will handle the styling and other things if needed.
See [this comment](https://github.com/vatesfr/xen-orchestra/pull/8142#discussion_r1858885424) for more explanation, even if it is for XO 6, the ~~song~~ concept remains the same.